examples/gallery/symbols/multi_parameter_symbols.py

```diff
@@ -26,37 +26,26 @@
 import pygmt
 
 fig = pygmt.Figure()
-
 fig.basemap(region=[0, 6, 0, 2], projection="x3c", frame=True)
 
-###################
 # ELLIPSE
 data = np.array([[0.5, 1, 45, 3, 1]])
-
```

Comment:
Because this issue https://github.com/GenericMappingTools/pygmt/issues/1021 was fixed, I believe it's possible to use 
```
data = [[0.5, 1, 45, 3, 1]]
```
instead of having to convert it to numpy.array.

Reply:
Yes, done in 188dadc.